src/style.css

```diff
@@ -1,3 +1,74 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header-logo {
+  display: flex;
+  padding-right: 243px;
```

Comment:
You have used a fixed padding value for `.header-logo` which may cause layout issues on different screen sizes. It's better to use relative units like percentages or use CSS Grid or Flexbox for layout arrangements. Fixed values should be avoided unless necessary.